db/seeds.rb

```diff
@@ -8,7 +16,7 @@
       email_address: hash.fetch('email', nil),
       enabled: hash.fetch('enabled', true),
       estate: hash.fetch('estate'),
-      id: hash.fetch('id'),
+      id: prison_uuid_mappings[path],
```

Comment:
The DB generated one takes precedence.  However, I think it should simply skip that record and alert the user that they need to run the generator, commit the updated uuid mapping and re-run.